### ISSUE TITLE
StaticBitmap tooltip does not display

### ISSUE THREAD

**Issue by bkVBC** [chriskiehl/re-wx#18]
Love the project, coming from react land into wx world has never been easier!

I'm having trouble getting the tooltip to show up on a `StaticBitmap`. Seems to work fine on `Block` and `Button`. I tested it alone and with other components, no luck getting it to display:
```
    element = wsx(
        [
            c.Frame,
            {"title": "Tooltip demo", "show": True},
            [
                c.Block,
                {"tooltip": "Testing tooltip block..."},
                [c.Button, {"label": "Button", "tooltip": "Test button tooltip"}],
                [c.StaticBitmap, {"uri": "help.png", "tooltip": "Test tooltip image"}],
            ],
        ]
```

**Comment by jamesdbrock** [comment]
This is fixed in PR #12

You can use my re-wx fork until the PR is merged. https://github.com/xc-jp/re-wx

**Comment by bkVBC** [comment]
Thank you, @jamesdbrock - I was just looking at your stuff. Appreciate you taking this forward. Looking forward to using the new components in your PR as well.
